bin/MadGraph5_aMCatNLO/cards/production/13TeV/EtaTToLL_0j_4f_NNPDF31nnlo_13TeV/EtaTToLL_0j_madgraph_4f_NNPDF31nnlo_13TeV_extramodels.dat

```diff
@@ -0,0 +1 @@
+topponium_ufo.tar.gz
```

Comment:
btw, i didn't see this model under https://cms-project-generators.web.cern.ch/cms-project-generators/, maybe it needs to be uploaded there